demo/kubernetes/rook-operator.yaml

```diff
@@ -2,12 +2,12 @@ kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
   name: rook-operator
-  namespace: default
```

Comment:
Doesn't the operator's ClusterRole need to be in the `default` namespace? Since the service account and cluster role are also in the `default` namespace. See https://github.com/rook/rook/pull/639/files#diff-ed82f2be2b6dba56a73c9bad423ff55bR67